Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [v7,
	3/3] arm64: dts: Update X-Gene standby GPIO controller DTS entries
From: Quan Nguyen <qnguyen@apm.com>
X-Patchwork-Id: 8338601
Message-Id: <1455714909-3027-4-git-send-email-qnguyen@apm.com>
To: linus.walleij@linaro.org, linux-gpio@vger.kernel.org,
	devicetree@vger.kernel.org, linux-arm-kernel@lists.infradead.org,
	Thomas Gleixner <tglx@linutronix.de>,
	Jason Cooper <jason@lakedaemon.net>, Marc Zyngier <marc.zyngier@arm.com>
Cc: Feng Kan <fkan@apm.com>, Quan Nguyen <qnguyen@apm.com>,
	Phong Vo <pvo@apm.com>, Duc Dang <dhdang@apm.com>, patches@apm.com,
	Loc Ho <lho@apm.com>, Iyappan Subramanian <isubramanian@apm.com>,
	Y Vo <yvo@apm.com>
Date: Wed, 17 Feb 2016 20:15:09 +0700
Patch-mainline: Submitted, https://patchwork.kernel.org/patch/8338601/

Update APM X-Gene standby GPIO controller DTS entries to enable it
as interrupt controller.

Signed-off-by: Y Vo <yvo@apm.com>
Signed-off-by: Quan Nguyen <qnguyen@apm.com>
Reviewed-by: Linus Walleij <linus.walleij@linaro.org>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>

---
arch/arm64/boot/dts/apm/apm-storm.dtsi | 3 +++
 arch/arm64/boot/dts/apm/apm-storm.dtsi |    3 +++
 1 file changed, 3 insertions(+)

--- a/arch/arm64/boot/dts/apm/apm-storm.dtsi
+++ b/arch/arm64/boot/dts/apm/apm-storm.dtsi
@@ -771,6 +771,9 @@
 					<0x0 0x2b 0x1>,
 					<0x0 0x2c 0x1>,
 					<0x0 0x2d 0x1>;
+			interrupt-parent = <&gic>;
+			#interrupt-cells = <2>;
+			interrupt-controller;
 		};
 
 		rtc: rtc@10510000 {
